service/controller/key/key.go

```diff
@@ -16,27 +16,27 @@ const (
 )
 
 func AppOperatorAppName(getter LabelsGetter) string {
-	return fmt.Sprintf("%s-app-operator", ClusterID(getter))
+	return fmt.Sprintf("app-operator-%s", ClusterID(getter))
 }
 
 func AppOperatorValuesResourceName(getter LabelsGetter) string {
-	return fmt.Sprintf("%s-app-operator-values", ClusterID(getter))
+	return fmt.Sprintf("app-operator-values-%s", ClusterID(getter))
 }
 
 func BaseDomain(getter LabelsGetter, base string) string {
 	return fmt.Sprintf("%s.%s", ClusterID(getter), base)
```

Comment:
Not changing this one as it ends up in the `cluster-values` config map as the domain that will be used.